packages/sanity/src/core/store/_legacy/history/history/Timeline.ts

```diff
@@ -59,14 +60,15 @@ export class Timeline {
   private _trace?: TraceEvent[]
 
   constructor(opts: TimelineOptions) {
-    this.publishedId = opts.publishedId
-    this.draftId = `drafts.${opts.publishedId}`
+    const {documentId} = opts
+    this.publishedId = documentId
```

Comment:
Do we still need this `publishedId`?

I'm thinking that maybe the answer is yes but then why not use this in the this._trace? (line 71) 👇